assets/src/scss/pages/_404.scss

```diff
@@ -1,12 +1,6 @@
 .page-404-page {
   .page-header {
-    color: $grey-80;
-    min-height: 1200px;
-    padding-top: 80px;
-
-    @include medium-and-up {
-      padding-top: 110px;
-    }
+    min-height: 100vh;
```

Comment:
This looks better on mobile than before, where `1200px` was really long (also seems to be a random value)